### PR TITLE
[CI:DOCS] Improve the description of fields in podman-stats man page

### PR DESCRIPTION
**Pull request by rhatdan**
Fixes: https://github.com/containers/podman/issues/15146

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: docs/source/markdown/podman-stats.1.md.in [hunk 1]
```diff
@@ -34,19 +34,19 @@ Valid placeholders for the Go template are listed below:
...
-| .BlockOutput        | Block Output                                     |
+| .BlockInput         | Total data read from block device                |
+| .BlockIO            | Total data read/total data written to block device|
+| .BlockOutput        | Total data written to block device               |
 | .ContainerID        | Container ID, full (untruncated) hash            |
 | .ContainerStats ... | Nested structure, for experts only               |
 | .CPU                | Percent CPU, full precision float                |
 | .CPUNano            | CPU Usage, total, in nanoseconds                 |
-| .CPUPerc            | CPU percentage                                   |
+| .CPUPerc            | Percentage of CPU Used                           |
```
Comment (vrothberg) [inline docs/source/markdown/podman-stats.1.md.in:44]:
```suggestion
| .CPUPerc            | Percentage of CPU used                           |
```

**Review by vrothberg — COMMENTED** [review]
Other than the typo LGTM
Please reformat the table on repush.

**Comment by rhatdan** [comment]
@TomSweeneyRedHat

**Comment by rhatdan** [comment]
@flouthoc @giuseppe @flouthoc @ashley-cui PTAL

File: docs/source/markdown/podman-stats.1.md.in [hunk 2]
```diff
@@ -34,19 +34,19 @@ Valid placeholders for the Go template are listed below:
 |---------------------|--------------------------------------------------|
 | .AvgCPU             | Average CPU, full precision float                |
 | .AVGCPU             | Average CPU, formatted as a percent              |
-| .BlockInput         | Block Input                                      |
-| .BlockIO            | Block IO                                         |
-| .BlockOutput        | Block Output                                     |
+| .BlockInput         | Total data read from block device                |
+| .BlockIO            | Total data read/total data written to block device|
```
Comment (flouthoc) [inline docs/source/markdown/podman-stats.1.md.in:38]: This row's last `|` is not in `sync` with other row, I mean this expands table horizontally. Just wondering if this is rendering fine.

**Review by flouthoc — APPROVED** [review]
LGTM

File: docs/source/markdown/podman-stats.1.md.in (hunk 2 above)
Comment (rhatdan) [reply docs/source/markdown/podman-stats.1.md.in:38]: It renders fine, and I dont think it is worth pushing.

**Pull request merged**